### ISSUE TITLE
Failed install of ImageServer-3.6.9.3 - Wrong parameters

### ISSUE THREAD

**Issue by coret** [Daniel-KM/Omeka-S-module-ImageServer#5]
When I tried to upgrade the Image Server to version 3.6.9.3 (on my Omeka S version 3.1.1 installation), the upgrade failed. 
I removed the old version and tried an install, that failed too, with the following error in the logs:

```
2022-02-13T14:57:41+00:00 ERR (3): Error: Wrong parameters for Omeka\Module\Exception\ModuleCannotInstallException([string $message [, long $code [, Throwable $previous = NULL]]]) in /home/**/omeka-s-3.1.1/modules/ImageServer/Module.php:120
Stack trace:
#0 /home/**/omeka-s-3.1.1/modules/ImageServer/Module.php(120): Exception->__construct()
#1 /home/**/omeka-s-3.1.1/modules/Generic/AbstractModule.php(63): ImageServer\Module->preInstall()
#2 /home/**/omeka-s-3.1.1/application/src/Module/Manager.php(240): Generic\AbstractModule->install()
#3 /home/**/omeka-s-3.1.1/application/src/Controller/Admin/ModuleController.php(117): Omeka\Module\Manager->install()
#4 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-mvc/src/Controller/AbstractActionController.php(77): Omeka\Controller\Admin\ModuleController->installAction()
#5 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Laminas\Mvc\Controller\AbstractActionController->onDispatch()
#6 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-eventmanager/src/EventManager.php(178): Laminas\EventManager\EventManager->triggerListeners()
#7 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-mvc/src/Controller/AbstractController.php(103): Laminas\EventManager\EventManager->triggerEventUntil()
#8 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-mvc/src/DispatchListener.php(139): Laminas\Mvc\Controller\AbstractController->dispatch()
#9 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Laminas\Mvc\DispatchListener->onDispatch()
#10 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-eventmanager/src/EventManager.php(178): Laminas\EventManager\EventManager->triggerListeners()
#11 /home/**/omeka-s-3.1.1/vendor/laminas/laminas-mvc/src/Application.php(331): Laminas\EventManager\EventManager->triggerEventUntil()
#12 /home/**/omeka-s-3.1.1/index.php(21): Laminas\Mvc\Application->run()
#13 {main}

```

**Comment by Daniel-KM** [comment]
Thanks, it is fixed in last version (3.6.10.3).

**Comment by coret** [comment]
3.6.10.3 installed OK!
